### PR TITLE
Feature/new switch camera

### PR DESCRIPTION
**Pull request by Julien-Sahli**

### CODE REVIEW

File: app/src/main/java/ch/epfl/sdp/MapActivity.kt [hunk 1]
```diff
@@ -384,6 +395,22 @@ class MapActivity : MapViewBaseActivity(), OnMapReadyCallback {
         }
     }
 
+
+    fun resizeFragment(v : View){
```
Comment (Ph0tonic) [inline app/src/main/java/ch/epfl/sdp/MapActivity.kt:399]: Should be rename to be more meaningful like `resizeCameraFragment` for example.

Comment (Julien-Sahli) [reply app/src/main/java/ch/epfl/sdp/MapActivity.kt:416]: Done !

File: app/src/main/java/ch/epfl/sdp/ui/vlc/VlcViewModel.kt [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class VlcViewModel : ViewModel() {
+
+    private val _text = MutableLiveData<String>().apply {
+        value = "This is VLC Fragment"
+    }
+    val text: LiveData<String> = _text
+}
```
Comment (Ph0tonic) [inline app/src/main/java/ch/epfl/sdp/ui/vlc/VlcViewModel.kt:13]: Do you really need this modelView to display a simple name ?

File: app/src/main/java/ch/epfl/sdp/MapActivity.kt [hunk 3]
```diff
@@ -1,496 +1,523 @@
...
+
+    /**
+     * Clears the waypoints list and removes all the lines and points related to waypoints
+     */
+    fun clearWaypoints(v: View) {
+        if (isMapReady) {
+            searchAreaBuilder.reset()
+        }
+    }
+
```
Comment (Ph0tonic) [inline app/src/main/java/ch/epfl/sdp/MapActivity.kt:414]: Remove this line

Comment (Julien-Sahli) [reply app/src/main/java/ch/epfl/sdp/MapActivity.kt:414]: Done

File: app/src/main/java/ch/epfl/sdp/ui/vlc/VlcViewModel.kt (hunk 2 above)
Comment (Julien-Sahli) [reply app/src/main/java/ch/epfl/sdp/ui/vlc/VlcViewModel.kt]: Actually not, in hindsight ^^

File: app/src/main/java/ch/epfl/sdp/ui/vlc/VlcFragment.kt [hunk 4]
```diff
@@ -1,74 +1,91 @@
...
+
+class VlcFragment : Fragment() {
+
+    companion object {
+        private const val USE_TEXTURE_VIEW = false
+        private const val ENABLE_SUBTITLES = false
+        private const val ASSET_FILENAME = "rtsp://192.168.1.131:8554/live" //must be your IP
+        private val ARGS = arrayListOf("-vvv")//, "--live-caching=200")
+    }
+
```
Comment (Ph0tonic) [inline app/src/main/java/ch/epfl/sdp/ui/vlc/VlcFragment.kt:27]: Remove this empty line

File: app/src/main/res/layout/fragment_vlc.xml [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+        <Button
+            android:id="@+id/switch_button"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="@string/switch_video"
+            app:layout_constraintBottom_toBottomOf="@+id/video_layout"
+            app:layout_constraintEnd_toEndOf="parent"
+            app:layout_constraintStart_toStartOf="parent" />
+
+
```
Comment (Ph0tonic) [inline app/src/main/res/layout/fragment_vlc.xml:26]: Remove those two empty lines

File: app/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -20,6 +20,7 @@
 
     <string name="menu_home">Home</string>
     <string name="menu_maps_managing">Maps managing</string>
+    <string name="menu_camera">Camera</string>
```
Comment (Ph0tonic) [inline app/src/main/res/values/strings.xml:23]: Seems to be unused

File: app/src/main/java/ch/epfl/sdp/ui/vlc/VlcFragment.kt (hunk 4 above)
Comment (Julien-Sahli) [reply app/src/main/java/ch/epfl/sdp/ui/vlc/VlcFragment.kt]: Done

File: app/src/main/res/layout/fragment_vlc.xml (hunk 5 above)
Comment (Julien-Sahli) [reply app/src/main/res/layout/fragment_vlc.xml]: Done

File: app/src/main/res/values/strings.xml (hunk 6 above)
Comment (Julien-Sahli) [reply app/src/main/res/values/strings.xml]: Removed

**Pull request merged**